specification/powerbiprivatelinks/resource-manager/Microsoft.PowerBI/stable/2020-06-01/powerbiprivatelinks.json

```diff
@@ -0,0 +1,1627 @@
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "description": "Deletes a private endpoint connection for Power BI by private endpoint name.",
+        "summary": "Asynchronous API to delete a private endpoint connection for Power BI by private endpoint name.",
+        "operationId": "PrivateEndpointConnections_Delete",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AzureResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation completed successfully."
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "type": "string",
+                "description": "The Azure async operation id to poll on result."
+              }
+            }
+          },
+          "204": {
+            "description": "No Content."
+          },
+          "default": {
+            "description": "Default Response. It will be deserialized as per the Error definition specified in the schema. Exception will be thrown.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Deletes private endpoint connection": {
+            "$ref": "./examples/PrivateEndpointConnections_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AsyncOperationDetail": {
+      "title": "AsyncOperationDetail",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The operation id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation name."
+        },
+        "status": {
+          "type": "string",
+          "description": "The operation status."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The operation start time."
+        },
+        "endTime": {
+          "type": "string",
+          "description": "The operation end time."
+        },
+        "error": {
+          "$ref": "#/definitions/Error",
+          "description": "The error."
+        }
+      }
+    },
+    "TenantResource": {
+      "title": "TenantResource",
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Consider $ref of common-types definitions for these standard properties.

Reply:
not sure what is the ask here

Reply:
reviewer is suggesting to make it a subtype of e.g. TrackedResource from https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json

Reply:
Leaving it as is a there are additional properties that are exposed.